### PR TITLE
gh-2672 Add redeem deadline warning

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #2672

Changes proposed in this pull request:
- Add redeem deadline warning

### CODE REVIEW

File: Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift [hunk 1]
```diff
@@ -44,6 +46,8 @@ class ClaimedAmountInputCell: UITableViewCell {
         descriptionLabel.text = "Select all tokens or custom amount."
         descriptionLabel.setStyle(.secondary)
 
+        redeemWarningLabel.setStyle(.footnote2.color(UIColor(hex: "#B2BBC0")))
```
Comment (biafra23) [inline Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift:49]: Should this be its own style? `smallprint` for example?

**Review by biafra23 — APPROVED** [review]
LGTM 🚀

File: Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift (hunk 1 above)
Comment (biafra23) [reply Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift:49]: It is used in two places already:
image

Comment (elgatovital) [reply Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift:49]: Label style is defined in figma as footnote2 with #B2BBC0 color. Let's revisit this later if we have more usages of this style and color combination. cc @TanyaEfremova

File: Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift [hunk 2]
```diff
@@ -36,6 +38,11 @@ class ClaimedAmountInputCell: UITableViewCell {
         fieldDelegate.isUsingMaxValue
     }
 
+    var redeemDeadlineWarning: Bool {
+        get { !redeemWarningLabel.isHidden }
```
Comment (MouazAlzahabi) [inline Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift]: We don't need that getter, I would remove it and use set(redeemDeadlineLabelVisible: Bool) method

**Review by MouazAlzahabi — CHANGES_REQUESTED** [review]

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**